test/components/admin/audit_trail_component_test.rb

```diff
@@ -0,0 +1,18 @@
+require "test_helper"
+
+class Admin::AuditTrailComponentTest < ViewComponent::TestCase
+  setup do
+    @non_editionable_item = create(:worldwide_organisation)
+  end
+
+  test "renders audit trail entries" do
+    create(:version, event: "create", item: @non_editionable_item)
+    create(:version, event: "update", item: @non_editionable_item)
+    versions = Kaminari.paginate_array(@non_editionable_item.versions_desc).page(1)
```

Comment:
As with the implementation in the controller, you should be able to avoid using `paginate_array` here.

While it's not the end of the world to use it here (it's never going to cause any performance issues..) it sets a bad example for how to implement this when using Kaminari.

```suggestion
    versions = @non_editionable_item.versions_desc.page(1)
```